### PR TITLE
Fixes Trailling Whitespaces In Strings

### LINKED ISSUE #4197: Trailing whitespace in i18n messages (+extra)
**Summary:** 

Translatewiki.net does not support trailing whitespace in messages (it gets stripped). Here is a list of such messages:
```
Wikimedia:Commons-android-strings-bullet/en
'•  '                                | '•'                                 

Wikimedia:Commons-android-strings-about license/en
'The Wikimedia Commons app is an       | 'The Wikimedia Commons app is an      
open-source app created and maintained | open-source app created and maintained
by grantees and volunteers of the      | by grantees and volunteers of the     
Wikimedia community. The Wikimedia     | Wikimedia community. The Wikimedia    
Foundation is not involved in the      | Foundation is not involved in the     
creation, development, or maintenance  | creation, development, or maintenance 
of the app. '                          | of the app.'                          

Wikimedia:Commons-android-strings-question/en
'Question '                            | 'Question'                            

Wikimedia:Commons-android-strings-achievements revert limit message/en
'minimum required: '                   | 'minimum required:'                   

Wikimedia:Commons-android-strings-bad token error proposed solution/en
'Upload failed due to issues with edit | 'Upload failed due to issues with edit
token. Please try logging out and in   | token. Please try logging out and in  
again. '                               | again.'                               

Wikimedia:Commons-android-strings-review c violation report question/en
'It is copyright violation because it  | 'It is copyright violation because it 
is '                                   | is'                                   

```

Insignificant trailing whitespace should be removed. Significant trailing whitespace should be avoided by using variables.

The script also found this difference:
```
Wikimedia:Commons-android-strings-upload image duplicate/en
'This file already destroyed on        | 'This file already exists on Commons. 
Commons. Are you sure you want to      | Are you sure you want to proceed?'    
proceed?'                              |                                       
```

The change from "exists" to "destroyed" looks like a mistake. Can you verify?

### PR DESCRIPTION
**Pull request by maxbetton**
**Description (required)**

Fixes #4197

---

_Note: Please ensure that you have read CONTRIBUTING.md if this is your first pull request._

### CODE REVIEW

**Review by madhurgupta10 — APPROVED** [review]

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -191,7 +191,7 @@
   <string name="no_nearby">No nearby places found</string>
   <string name="warning">Warning</string>
   <string name="duplicate_image_found">Duplicate Image Found</string>
-  <string name="upload_image_duplicate">This file already destroyed on Commons. Are you sure you want to proceed?</string>
+  <string name="upload_image_duplicate">This file already exists on Commons. Are you sure you want to proceed?</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml:194]: Hey @maxbetton this change seems irrelevant, correct me if I am wrong. If it has been made by mistake, can you please fix it?

**Review by neslihanturan — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (maxbetton) [reply app/src/main/res/values/strings.xml:194]: reffering to issue #4197 I think i'm right

**Review by neslihanturan — APPROVED** [review]
Thanks and congratz for first contribution :partying_face:
